### PR TITLE
Change logging configuration set at package initialization

### PR DESCRIPTION
**Pull request by rouille**
### Purpose
Change OGE logger configuration set at package initialization. The default was to print logs to the console, and to a file. Now it only prints to the console. Users can overwrite this configuration in their own project if it is desired. 

### What the code is doing
Set the `logfile` argument to `None` when calling `configure_root_logger` in \_\_init\_\_.py file.

### Testing
It has been tested in an external project importing `oge`.

### Where to look
In the \_\_init\_\_.py file.

### Usage Example/Visuals
N/A

### Review estimate
2min

### Future work
N/A

### Checklist
- [x] Update the documentation to reflect changes made in this PR
- [x] Format all updated python files using `black`
- [x] Clear outputs from all notebooks modified
- [x] Add docstrings and type hints to any new functions created

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,14 +95,14 @@ All files downloaded/created as part of the pipeline are stored in your HOME dir
...
-Note that you don't need to run the pipeline to generate the output data as these are available on Amazon Simple Storage Service (S3). Simply, set the `OGE_DATA_STORE` environment variable to `s3` in the **\_\_init\_\_.py** file of your project to fetch OGE data from Amazon S3. Additionally, you can tell OGE to not write logs in a file by overwriting the default configuration of the OGE logger.
+Note that you don't need to run the pipeline to generate the output data as these are available on Amazon Simple Storage Service (S3). Simply, set the `OGE_DATA_STORE` environment variable to `s3` in the **\_\_init\_\_.py** file of your project to fetch OGE data from Amazon S3. Additionally, you can tell OGE to write logs in a file by overwriting the default configuration of the OGE logger.
 
 To summarize, your **\_\_init\_\_.py** file would then look like this:
 ```python
 from oge.logging_util import configure_root_logger
 
 os.environ["OGE_DATA_STORE"] = "s3"
-configure_root_logger(logfile=None)
+configure_root_logger(logfile="oge_log/logfile.txt")
```
Comment (grgmiller) [inline README.md:105]: Shouldn't we just delete this line now since the default setting will be for OGE to not save logs unless running the pipeline, which is what we want?

I would just delete "Additionally, you can tell OGE to write logs in a file by overwriting the default configuration of the OGE logger." and line 105

**Review by grgmiller — APPROVED** [review]
See below, I'd suggest that we just delete references to teh logger configuration since we're now fixing the default behavior and shouldn't need to add anything to the project's init file about logging.

File: README.md (hunk 1 above)
Comment (rouille) [reply README.md:105]: Will do so. This was intended for external projects who wants to save OGE logs

Comment (rouille) [reply README.md]: Done in [97c4c35](https://github.com/singularity-energy/open-grid-emissions/pull/340/commits/97c4c35819bc3eb159334ec26a988efc40cd842e)

File: README.md [hunk 2]
```diff
@@ -95,14 +95,10 @@ All files downloaded/created as part of the pipeline are stored in your HOME dir
...
 OGE is not yet available on PyPi but can be installed from GitHub. For example, this can be done by adding `oge = {git="https://github.com/singularity-energy/open-grid-emissions.git"}` to your Pipfile if you are using `pipenv` for your project.
 
-Note that you don't need to run the pipeline to generate the output data as these are available on Amazon Simple Storage Service (S3). Simply, set the `OGE_DATA_STORE` environment variable to `s3` in the **\_\_init\_\_.py** file of your project to fetch OGE data from Amazon S3. Additionally, you can tell OGE to not write logs in a file by overwriting the default configuration of the OGE logger.
-
+Note that you don't need to run the pipeline to generate the output data as these are available on Amazon Simple Storage Service (S3). Simply, set the `OGE_DATA_STORE` environment variable to `s3` in the **\_\_init\_\_.py** file of your project to fetch OGE data from Amazon S3.
 To summarize, your **\_\_init\_\_.py** file would then look like this:
 ```python
-from oge.logging_util import configure_root_logger
-
 os.environ["OGE_DATA_STORE"] = "s3"
```
Comment (grgmiller) [inline README.md:101]: One other comment on this that I just noticed - would the user also need to `import os` as well here?

**Review by grgmiller — APPROVED** [review]
One additional small change, but otherwise looks good

File: README.md (hunk 2 above)
Comment (rouille) [reply README.md:103]: Done

Comment (grgmiller) [reply README.md:103]: Thanks, looks good to merge
